### ISSUE TITLE
[Feature] Active level debug only for a task

### ISSUE THREAD

**Issue by melck** [influxdb/kapacitor#121]
I would like to set log level at debug only for a task. I have multiple inputs and its difficult to read it all logs .
May be implements a new command to see logs of specific tasks ?

**Comment by nathanielc** [comment]
I agree the logging needs to be configurable by task. For now every line related to a task should have the task name in the line. As a result you should be able to grep for only the logs for a specific task.

**Comment by jommgoncalves** [comment]
+1
